webapp/src/shared/components/Input/Input.tsx

```diff
@@ -0,0 +1,41 @@
+import { Color } from '../../types';
+import Icon, { IconSize, IconVariant } from '../Icon/Icon';
+import './Input.css';
+
+export enum InputColor {
+  LIGHT = 'light',
+  DARK = 'dark',
+}
+
+type InputProps = {
```

Comment:
question: What do we think about adding a `value` and `onChange` optional props to allow creating controlled inputs?

Then we could simply pass them to the input tag:
```
<input ... onChange={onChange} value={value} />
```

Reply:
Agree with @gbudau and another option for use could be [useRef](https://es.reactjs.org/docs/hooks-reference.html#useref) instead of names. Names could end up being problematic if we have inputs with the same name, but ref identifies them unequivocally